bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html

```diff
@@ -85,7 +89,9 @@ <h3>{{ _('Knowledge that protects') }}</h3>
         <img class="glyphs" src="{{ static('img/firefox/whatsnew_67/heart_glyph.svg') }}" alt="">
         <div class="benefit-content">
           <h3>{{ _('Fierce defense of your privacy') }}</h3>
-          <p>{{ _('Everything we make and do honors our Personal Data Promise: <strong>Take less. Keep it safe. No secrets.</strong>') }}</p>
+          <p>{{ _('Everything we make and do honors our Personal Data Promise:') }}
```

Comment:
This will also break translations. The original string had no `<strong>` tag so it became a mismatch when that was added. Now it's being split, so it's two new strings, neither of which are translated. We need to either remove the `<strong>` to revert back to the original string, or update the lang files. 

Since I know marketing will want the emphasis, and since I think we have to update the lang files for the other string change above already, I think we should do this too.

https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/whatsnew_67.0.5.lang#L99

Reply:
And I also think splitting this into two strings is better than one, so this is also good.